pkg/storage/migrations/source_schema.go

```diff
@@ -0,0 +1,12 @@
+package migrations
+
+import (
+	"github.com/cnabio/cnab-go/schema"
+)
+
+// SourceSchema represents the file format of Porter's v0.38 schema document
+type SourceSchema struct {
+	Claims      schema.Version `json:"claims"`
```

Comment:
Just curious, why does this schema only supports `json` format vs `SourceParameterSet`

Reply:
These source data structs are copied directly from porter v0.38. The schema document only ever supported json, while parameter sets and credential sets supported json and yaml (since we read them from the user on commands like porter install). Porter v0.38 only ever stored parameter and credential sets as json files though when using the old filesystem storage plugin.